go.mod

```diff
@@ -85,7 +85,7 @@ require (
 	knative.dev/pkg v0.0.0-20230221145627-8efb3485adcf // indirect
 	sigs.k8s.io/controller-runtime v0.12.3
 	sigs.k8s.io/controller-tools v0.9.2
-	sigs.k8s.io/prow v0.0.0-20240409200154-0bca2f1416a9
+	sigs.k8s.io/prow v0.0.0-20240419142743-3cb2506c2ff3
```

Comment:
Does Prow do releases?

Reply:
The repo was just recently created, so there's no precedence for releases yet. Not sure if this will change.

Reply:
Would be interesting to sort out, but not a blocker for this PR.

/lgtm